routes/web.php

```diff
@@ -89,28 +89,6 @@
 Route::get('conferences', ['as' => 'conferences.index', 'uses' => 'ConferencesController@index']);
 Route::get('conferences/{id}', ['as' => 'conferences.show', 'uses' => 'ConferencesController@show']);
 
```

Comment:
Are these not in use anymore? I remember something happening with these but surprised to see in this PR